_commands/tcpdump.md

```diff
@@ -0,0 +1,101 @@
+22:59:52.311976 IP bud02s22-in-f196.1e100.net.https > 10.0.2.15.59170: Flags [P.], seq 110071562:110081572, ack 2068573116, win 65535, length 10010
+22:59:52.311983 IP 10.0.2.15.59170 > bud02s22-in-f196.1e100.net.https: Flags [.], ack 110081572, win 65535, length 0
+22:59:52.408516 IP bud02s22-in-f196.1e100.net.https > 10.0.2.15.59170: Flags [P.], seq 110081572:110084203, ack 2068573116, win 65535, length 2631
+22:59:52.408532 IP 10.0.2.15.59170 > bud02s22-in-f196.1e100.net.https: Flags [.], ack 110084203, win 65535, length 0
+22:59:52.408559 IP bud02s22-in-f196.1e100.net.https > 10.0.2.15.59170: Flags [P.], seq 110084203:110085633, ack 2068573116, win 65535, length 1430
+22:59:52.408564 IP 10.0.2.15.59170 > bud02s22-in-f196.1e100.net.https: Flags [.], ack 110085633, win 65535, length 0
+22:59:52.415046 IP bud02s22-in-f196.1e100.net.https > 10.0.2.15.59170: Flags [P.], seq 110085633:110091353, ack 2068573116, win 65535, length 5720
+22:59:52.415063 IP 10.0.2.15.59170 > bud02s22-in-f196.1e100.net.https: Flags [.], ack 110091353, win 65535, length 0
+22:59:52.415089 IP bud02s22-in-f196.1e100.net.https > 10.0.2.15.59170: Flags [P.], seq 110091353:110092458, ack 2068573116, win 65535, length 1105
+22:59:52.415299 IP 10.0.2.15.59170 > bud02s22-in-f196.1e100.net.https: Flags [P.], seq 2068573116:2068573162, ack 110092458, win 65535, length 46
+22:59:52.415515 IP bud02s22-in-f196.1e100.net.https > 10.0.2.15.59170: Flags [.], ack 2068573162, win 65535, length 0
+22:59:52.820267 IP 10.0.2.15.59170 > bud02s22-in-f196.1e100.net.https: Flags [P.], seq 2068573162:2068573471, ack 110092458, win 65535, length 309
+22:59:52.820852 IP bud02s22-in-f196.1e100.net.https > 10.0.2.15.59170: Flags [.], ack 2068573471, win 65535, length 0
+22:59:52.970387 IP bud02s22-in-f196.1e100.net.https > 10.0.2.15.59170: Flags [P.], seq 110092458:110092543, ack 2068573471, win 65535, length 85
+22:59:52.972855 IP bud02s22-in-f196.1e100.net.https > 10.0.2.15.59170: Flags [P.], seq 110092543:110092589, ack 2068573471, win 65535, length 46
+22:59:52.973051 IP 10.0.2.15.59170 > bud02s22-in-f196.1e100.net.https: Flags [.], ack 110092589, win 65535, length 0
+22:59:52.973194 IP 10.0.2.15.59170 > bud02s22-in-f196.1e100.net.https: Flags [P.], seq 2068573471:2068573517, ack 110092589, win 65535, length 46
+22:59:52.973558 IP bud02s22-in-f196.1e100.net.https > 10.0.2.15.59170: Flags [.], ack 2068573517, win 65535, length 0
+23:00:02.046795 IP 10.0.2.15.56854 > bud02s22-in-f196.1e100.net.http: Flags [.], ack 109761678, win 31844, length 0
+23:00:02.047116 IP bud02s22-in-f196.1e100.net.http > 10.0.2.15.56854: Flags [.], ack 1151493304, win 65535, length 0
+23:00:12.062722 IP 10.0.2.15.56854 > bud02s22-in-f196.1e100.net.http: Flags [.], ack 109761678, win 31844, length 0
+23:00:12.062908 IP bud02s22-in-f196.1e100.net.http > 10.0.2.15.56854: Flags [.], ack 1151493304, win 65535, length 0
+~~~
+##### Break it down
+-The -S flag prints absolute TCP sequence numbers versus relative
```

Comment:
Surround inline code (including command like options like `-S`) in \`backticks\` so they appear `like this`.